pkg/util/test/util.go

```diff
@@ -2928,6 +2934,171 @@ func ValidatePrometheus(pxImageList map[string]string, cluster *corev1.StorageCl
 	return nil
 }
 
+func ValidateGrafana(cluster *corev1.StorageCluster) error {
+	opVersion, err := GetPxOperatorVersion()
+	if err != nil {
+		return err
+	}
+	if opVersion.LessThanOrEqual(opVer23_8) {
```

Comment:
This will skip validation even if your PX Operator version is 23.8, I thought you wanted to validate this starting with 23.8?